regenerate_vscode.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Allow version control of vscode settings at .vscode_shared/UserName, without having defaults brought in on updates (i.e. don't track .vscode)
+# Not sure bash is good for everyone, they can adjust as needed.
+
+if [ "$#" -ne 1 ]; then
+    echo "WARNING: you need to specify a users name FirstLast"
+    exit 1
+fi
+
+NAME=${1}
+
+case $NAME in
+    "AlexisRohou") 
+        ;;
+    "BenHimes")
+        ;;
+    *)
+        echo "WARNING, the Name $NAME is not recognized"
+        exit 1
+esac
+
+cd .vscode
+echo "Linking vscode settings for user $NAME"
+ls ../.vscode_shared/${NAME} | while read FILE_NAME ; do
+    echo $FILE_NAME
+    ln -sf ../.vscode_shared/${NAME}/$FILE_NAME $(basename $FILE_NAME)
+    done
```

Comment:
I have a question: why don't we just each have a local `.vscode` symlink to `.vscode_shared/${NAME}`. This would be a static symlink, which we could add to .gitignore so that we don't overwrite each others'. 
Advantage of this approach, if it works, would be that there'd be no need for a script to be run and maintained.

Reply:
Great idea -